app/uk/gov/hmrc/brm/utils/BRMFormat.scala

```diff
@@ -30,9 +31,17 @@ trait BRMFormat {
 
   private val invalidNameCharsRegEx =  "[;/\\\\(){}&|]|(<!)|(-->)|(\\n)|(\")".r
   private val validBirthReferenceNumberRegEx = """^[a-zA-Z0-9_-]+$"""
```

Comment:
@adamconder This is still used on the Payload validation to catch invalid characters. The same regex is used on the check for length so it could be removed if you think it's just duplication? (my thought process was if we can validate against Payload then do it and return any failed validation earlier)